apps/nsqd/options.go

```diff
@@ -133,6 +133,7 @@ func nsqdFlagSet(opts *nsqd.Options) *flag.FlagSet {
 	flagSet.String("tcp-address", opts.TCPAddress, "<addr>:<port> to listen on for TCP clients")
 	flagSet.Bool("use-unix-sockets", opts.UseUnixSockets, "Usinx UNIX sockets instead of IP sockets")
 	flagSet.String("wakeup-socket-dir", opts.WakeupSocketDir, "Directory of sockets to wake up the consumer processes")
+	flagSet.String("pidfile", opts.PIDFile, "Pidfile of current process")
```

Comment:
I suggest to keep context here (why we need pid file). Actually it neede only for legacy and could be removed right after when centos 6 would be unsupported, i.e. I suggest to use line `Pidfile of current process (centos 6 daemonization requirement)`